dev-python/paho-mqtt/metadata.xml

```diff
@@ -1,7 +1,14 @@
 <?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Please make sure to keep all the footer tags together (`Closes`+`Package-Manager`) in one block. I can fix this for you.